src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let money = days * 40;
+
+  if (days >= 7) {
+    money = money - 50;
+  }
+
+  if (days >= 3 && days < 7) {
+    money = money - 20;
+  }
+
+  return money;
```

Comment:
The function name 'calculateRentalCost' is descriptive, which is good. However, the code contains literals in expressions, which should be replaced with named constants for better readability and maintainability. For instance, instead of using '40', '50', and '20', you should define constants that describe what these values represent. Also, consider using 'const' for the 'money' variable since its value is not reassigned but only modified.